### ISSUE TITLE
Genbank Support

### ISSUE THREAD

**Issue by garethrees** [bioruby/bioruby#66]
I'm working with @catfeet to write a Blast pipeline.

The tool used at Cardiff University is [Nucleotide BLAST](http://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&BLAST_PROGRAMS=megaBlast&PAGE_TYPE=BlastSearch&SHOW_DEFAULTS=on&LINK_LOC=blasthome) with the nr/nt database from [Genbank](http://www.ncbi.nlm.nih.gov/genbank).

It seems like the only options with bioruby are `genomenet` and `ddbj`. However, `genomenet.rb`references `http://www.ncbi.nlm.nih.gov/blast/` in the notes.

Basically we want to be able to do:

``` ruby
blast = Bio::Blast.remote 'blastn', 'nr-nt', '-e 0.05 -m 8', 'genbank'
```

Does this mean I'll have to write a `Bio::Blast::Remote::Genbank` module to receive output from that tool?
